bash/ostree

```diff
@@ -399,6 +399,37 @@ _ostree_admin_os_init() {
     return 0
 }
 
+_ostree_admin_pin() {
+    local boolean_options="
+        $main_boolean_options
```

Comment:
`--unpin` ?

Reply:
I honestly don't grasp bash completion, I don't care too much about it myself at least for `ostree admin`.  Let's not try to polish it for now?

Reply:
There has to be someone out there who has implemented actually generating this stuff from the cli `--help`, as is it's an enormous maintenance pain doubled by the fact that we need to wrap it in `rpm-ostree` at least too.